src/Model/Changelog/ChangelogItem.php

```diff
@@ -59,7 +59,9 @@ class ChangelogItem
     protected static $types = array(
         'Security' => array(
             // E.g. "[ss-2015-016]: Security fix" - deliberately case insensitive here
-            '/^(\[SS-2(\d){3}-(\d){3}\]):?/i'
+            '/^(\[SS-2(\d){3}-(\d){3}\]):?/i',
+            // E.g. "[cve-2019-12345]: Security fix"
+            '/^(\[CVE-(\d){4}-(\d){4,5}\]):?/i',
```

Comment:
It is possible that it could reach 6 digits in a year. Do you think we can just drop the `5` here?

```suggestion
            '/^(\[CVE-(\d){4}-(\d){4,}\]):?/i',
```

Reply:
Yeah to be fair, [this spec](https://github.com/CVEProject/CVE-ID-Allocation-Service/blob/master/CVE_IAS_Spec.md) example uses 6 characters. Will update.